### PR TITLE
Refactor handlers for stopPropagation & PreventDefault

### LINKED ISSUE #258: Add docstrings
These functions need docstrings:

- [ ] stopPropagation
- [ ] preventDefault
- [ ] stopPropagationAndPreventDefault

### PR DESCRIPTION
**Pull request by p-netm**
closes #258

### CODE REVIEW

File: src/helpers/utils.tsx [hunk 1]
```diff
@@ -398,17 +398,21 @@ export function roundToPrecision(n: number, precision: number = 0): number {
...
 
-export function stopPropagation(e: Event | MouseEvent | any) {
-  e.stopPropagation();
-}
-
+/** handler that cancels an event's default behavior
+ *
+ * @param {Event | MouseEvent} e - a synthetic event wrapper around native dom events
+ */
 export function preventDefault(e: Event | MouseEvent | any) {
```
Comment (moshthepitt) [inline src/helpers/utils.tsx]: It seems that this one is not used anywhere?

File: src/containers/pages/InterventionPlan/IRS/plan/index.tsx [hunk 2]
```diff
@@ -48,7 +45,6 @@ import {
 } from '../../../../../constants';
 import {
   FlexObject,
-  preventDefault,
   RouteParams,
   stopPropagation,
```
Comment (moshthepitt) [inline src/containers/pages/InterventionPlan/IRS/plan/index.tsx]: Is this one still needed?

File: src/containers/pages/InterventionPlan/IRS/plan/index.tsx [hunk 3]
```diff
@@ -741,7 +737,7 @@ class IrsPlan extends React.Component<
   // Jurisdiction Hierarchy Control
   /** onTableBreadCrumbClick - handler for drilldown table breadcrumb clicks to reset the table hierarchy */
   private onTableBreadCrumbClick = (e: MouseEvent) => {
-    preventDefault(e);
+    e.preventDefault();
```
Comment (cKellyDesign) [inline src/containers/pages/InterventionPlan/IRS/plan/index.tsx]: This is pretty low hanging fruit, if this is too much hassle then let's just not have util functions for these.

File: src/helpers/utils.tsx (hunk 1 above)
Comment (p-netm) [reply src/helpers/utils.tsx]: yeah the idea was to only leave the one that has them both as a standalone function.
Dont think i was sober enough during this pr

File: src/containers/pages/InterventionPlan/IRS/plan/index.tsx (hunk 2 above)
Comment (p-netm) [reply src/containers/pages/InterventionPlan/IRS/plan/index.tsx]: yes,

File: src/containers/pages/InterventionPlan/IRS/plan/index.tsx (hunk 3 above)
Comment (p-netm) [reply src/containers/pages/InterventionPlan/IRS/plan/index.tsx]: `stopPropagation` and `stopPropagationAndPreventDefault` have to be click handler functions for the syntax to work:
```
```

Comment (cKellyDesign) [reply src/containers/pages/InterventionPlan/IRS/plan/index.tsx]: In the second example, I think we either replace `e.preventDefault()` with `preventDefault(e)` or we can just leave it as is.

Comment (p-netm) [reply src/containers/pages/InterventionPlan/IRS/plan/index.tsx]: ok.

File: src/containers/pages/InterventionPlan/IRS/plan/index.tsx [hunk 4]
```diff
@@ -817,7 +817,7 @@ class IrsPlan extends React.Component<
   }
   /** onSaveEditNameButtonClick - handler disabling inline-editing */
   private onSaveEditNameButtonClick(e: MouseEvent) {
-    e.preventDefault();
+    preventDefault();
```
Comment (cKellyDesign) [inline src/containers/pages/InterventionPlan/IRS/plan/index.tsx]: We'll need to pass `e` into `preventDefault` here

Comment (p-netm) [reply src/containers/pages/InterventionPlan/IRS/plan/index.tsx]: oh yeah. :+1:

**Comment by cKellyDesign** [comment]
I think the main/original use case for having these functions is feeding them directly into component handler attributes:

```
<Button onClick={preventDefault} />
```
which was intended to be a bit DRYer than:
```
function handleSpecificButtonClick(e) { e.preventDefault() }
<Button onClick={handleButtonClick} />
```
and to avoid the linting error `Lambdas are forbidden in JSX attributes due to their rendering performance impact` when doing:
```
<Button onClick={(e) => { e.preventDefault() } />
```

**Comment by cKellyDesign** [comment]
@moshthepitt thoughts on this? If you feel it's just excess overhead then let's scrap it and close #258

**Pull request merged**